exotic.py

```diff
@@ -407,30 +367,108 @@ def user_input(prompt, type_, val1=None, val2=None):
+                     "Published Time of Mid-Transit (BJD_UTC)",
+                     "Time of Mid-Transit Uncertainty (JD)",
+                     "Ratio of Planet to Stellar Radius (Rp/Rs)",
+                     "Ratio of Distance to Stellar Radius (a/Rs)",
+                     "Orbital Inclination (deg)",
+                     "Orbital Eccentricity (0 if null)",
+                     "Star Effective Temperature (K)",
+                     "Star Effective Temperature Positive Uncertainty (K)",
+                     "Star Effective Temperature Negative Uncertainty (K)",
+                     "Star Metallicity ([FE/H])",
+                     "Star Metallicity Uncertainty Positive ([FE/H])",
+                     "Star Metallicity Uncertainty Negative ([FE/H])",
+                     "Star Surface Gravity (log(g))",
+                     "Star Surface Gravity Uncertainty Positive (log(g))"
+                     "Star Surface Gravity Uncertainty Negative (log(g))"]
+
+    # Exoplanet confirmed in NEA
+    if CandidatePlanetBool:
+        print('Here are the values scraped from the NASA Exoplanet Archive for ' + pDict['pName'])
+        print('For each planetary parameter, enter "y" if you agree and "n" if you disagree')
+
+        for i, key in enumerate(pDict):
+            if key in ('pName', 'sName', 'ra', 'dec', 'flag'):
+                continue
+            print('\n' + targetName + ' ' + planet_params[i] + ': ' + str(pDict[key]))
+            agreement = user_input('Do you agree? (y/n): ', type_=str, val1='y', val2='n')
+            if agreement.lower() == 'y':
+                continue
+            elif agreement.lower() == 'n':
+                pDict[key] = user_input('Enter the ' + planet_params[i] + ': ', type_=float)
+
+    # Exoplanet not confirmed in NEA
+    else:
+        pDict = {'pName': None, 'sName': None, 'ra': None, 'dec': None, 'pPer': None, 'pPerUnc': None, 'midT': None,
+                 'midTUnc': None, 'rprs': None, 'aRs': None, 'inc': None, 'ecc': None, 'teff': None, 'teffUnc': None,
+                 'met': None, 'metUnc': None, 'logg': None, 'loggUnc': None}
+
+        for i, key in enumerate(pDict):
+            if key in ('pName', 'sName'):
+                pDict[key] = user_input('Enter the ' + planet_params[i] + ': ', type_=str)
+            elif key == 'ra':
+                    raStr = input('Enter the ' + planet_params[i] + ': ')
+                    pDict['ra'] = astropy.coordinates.Angle(raStr + " hours").deg
+            elif key == 'dec':
+                    decStr = input('Enter the ' + planet_params[i] + ': ')
+                    decStr = decStr.replace(' ', '').replace(':', '')
+                    pDict['dec'] = astropy.coordinates.Angle(decStr + " degrees").deg
+            else:
+                pDict[key] = user_input('Enter the ' + planet_params[i] + ': ', type_=float)
+
+    return pDict
+
```

Comment:
Added this portion of the code to make life easier when checking against the NEA or user input. Saved coding space and will make it easier to write to the parameters to init.txt file later on.